bin/run.sh

```diff
@@ -134,7 +131,6 @@ testAll() {
   coreDataTest
   commandTest
   metaDataTest
-  loggingTest
```

Comment:
Is that because Redis doesn't need `loggingTest`? Mongo still needs it, so I don't think  it should be removed now.

Reply:
@andresrinivasan although we run logging with file as persistence, we still need to run logging test to make sure the API works.
Theoretically, logging service APIs can work with file